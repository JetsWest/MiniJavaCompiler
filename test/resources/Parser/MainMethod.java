class M {
    public static void main(String[] args) {
        int a;
        int b;
        a = 1;
        b = 2;
        System.out.println(a + b);
    }
}