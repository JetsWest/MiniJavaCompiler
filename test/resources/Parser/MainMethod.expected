Syntax error for input symbol "INT" spanning from unknown:3/9(-1) to unknown:3/11(-1)
Couldn't repair and continue parse for input symbol "INT" spanning from unknown:3/9(-1) to unknown:3/11(-1)
